### PR TITLE
fix inconsistent license info; update readme

### PR DESCRIPTION
**Pull request by jhanca-robotecai**
- fix inconsistent license info
- update readme
- remove execute rights of collider files

### CODE REVIEW

File: Assets/Gripper/BasicMeshes/Gripper_collider.stl
Comment (arturkamieniecki) [inline Assets/Gripper/BasicMeshes/Gripper_collider.stl:1]: An empty file. Probably added by accident.

File: Assets/Gripper/BasicMeshes/Gripper_collider.stl.assetinfo
Comment (arturkamieniecki) [inline Assets/Gripper/BasicMeshes/Gripper_collider.stl.assetinfo:1]: An empty file. Probably added by accident.

**Review by arturkamieniecki — APPROVED** [review]
There seem to be some unnecessary files added with this PR.

File: Assets/Gripper/BasicMeshes/Gripper_collider.stl
Comment (jhanca-robotecai) [reply Assets/Gripper/BasicMeshes/Gripper_collider.stl:1]: It is not an empty file - Github tells you I changed permissions `100755 → 100644`.

**Pull request merged**